File: java/soma_vetor.java
import java.util.Scanner;
import java.util.Locale;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int N, i;
		
		System.out.print("Quantos numeros voce vai digitar? ");
		N = sc.nextInt();
		
		double[] vetor = new double[N];
		
		for (i = 0; i < N; i++) {
			System.out.print("Digite um numero: ");
			vetor[i] = sc.nextDouble();
		}
		
		double soma = 0;
		System.out.println();
		System.out.print("VALORES = ");
		
		for (i = 0; i < N; i++) {
			System.out.print(String.format("%.1f", vetor[i]) + " ");
			soma += vetor[i];
		}
		
		double media = soma/N;
		
		System.out.println();
		System.out.println("SOMA = " + String.format("%.2f", soma));
		System.out.println("MEDIA = " + String.format("%.2f", media));
		
		sc.close();
	}

}

File: python/soma_vetor.py
n = int(input("Quantos numeros voce vai digitar? "))

vetor: [float] = [0 for x in range(n)]

for i in range(0, n):
    vetor[i] = float(input("Digite um numero: "))

soma = 0

print("VALORES = ", end="")
for i in range(0, n):
    print("{:.1f} ".format(vetor[i]), end="")

print()

for i in range(0, n):
    soma += vetor[i]

media = soma/n

print(f"SOMA = {soma:.2f}")
print(f"MEDIA = {media:.2f}")




